.github/workflows/pr-lint.yml

```diff
@@ -51,16 +51,3 @@ jobs:
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           report-json: ./eslint-report.json
-
-
-      - name: Prettier
-        continue-on-error: true
-        id: prettier
-        shell: bash
-        run: |
-          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
-            if [[ $file == *.js || $file == *.ts || $file == *.jsx || $file == *.tsx || $file == *.mdx || $file == *.scss || $file == *.json ]]; then
```

Comment:
this is not needed because lint will show annotation on modified file that do not respect prettier rules